src/main/java/com/CoachPlan/dto/CoachDTO.java

```diff
@@ -3,13 +3,26 @@
 public class CoachDTO {
```

Comment:
Since these are in a DTO package, you can drop the DTO name from the class... but this is fairly minor.